integration-test-groups/aws2/aws2-sqs/src/test/java/org/apache/camel/quarkus/component/aws2/sqs/it/Aws2SqsTest.java

```diff
@@ -160,35 +160,35 @@ public void sqsAutoCreateDelayedQueue() {
     }
 
     private void createDelayQueueAndVerifyExistence(String queueName, int delay) {
-        RestAssured.post("/aws2-sqs-sns/sqs/queue/autocreate/delayed/" + queueName + "/" + delay)
+        RestAssured.post("/aws2-sqs/queue/autocreate/delayed/" + queueName + "/" + delay)
                 .then()
                 .statusCode(200)
                 .extract()
                 .body()
                 .as(String[].class);
-        Awaitility.await().pollInterval(1, TimeUnit.SECONDS).atMost(120, TimeUnit.SECONDS).until(
+        Awaitility.await().pollInterval(1, TimeUnit.SECONDS).atMost(10, TimeUnit.SECONDS).until(
```

Comment:
It looks the sqs subset of sqs-sns to take far less time. What is the reason of narrowing down the timeout from 120 to 10 ? Do we save that much time ?

Reply:
Oh, it was my local stuff. I need to put everything back as normal for tests against AWS.
Thanks for catching this

Reply:
fixed !!